File: Camera/app/src/main/java/com/example/ravneet/camera/MainActivity.java
package com.example.ravneet.camera;

import android.hardware.Camera;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    Camera c;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        c = Camera.open(0);
    }
}

File: TodoManager/app/src/main/java/com/example/ravneet/todomanager/MainActivity.kt
package com.example.ravneet.todomanager

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import kotlinx.android.synthetic.main.activity_main.*;
import java.util.*

class MainActivity : AppCompatActivity() {

    var toods:ArrayList<Todo>

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
